Examples.sln

```diff
@@ -51,6 +51,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTesting", "testing-unit
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.Functions.LinuxApp", "azure-cs-functions-linux-app\Azure.Functions.LinuxApp.csproj", "{B3739A06-5343-4990-A48B-E5BEFAC5678A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.CredentialRotation.OneSet", "azure-nextgen-cs-credential-rotation-one-set\Azure.CredentialRotation.OneSet.csproj", "{34988495-6782-4AE3-9224-DE26A8FA1C61}"
```

Comment:
Did you want this added to the solution? None of the other NextGen projects are included. Also, should the project name be renamed with AzureNextGen?

Reply:
Yes, I think we should add all NextGen projects eventually